### ISSUE TITLE
SemVer not compliant package

### ISSUE THREAD

**Issue by koresar** [center-key/clabe-validator#37]
Hello. @dpilafian 

Thank you for the awesome package!

Version 1.5.2 is not compatible with 1.5.1. You published a breaking change as SemVer MINOR.

The main `clabe.js` 1.5.1 used to have this line: 
```js
if (typeof module === "object") module.exports = clabe;
```

The 1.5.2 does not have it any more. It broke our production builds.

See this line: https://github.com/center-key/clabe-validator/commit/676263283285ee3f54ecf874415cb135baae4623#diff-ecf689c52ec37d55479746c939f7e7e13882c8df8848ce65741f8a1e1ad7ae0bL1085

Could you please add back the CommonJS compatibility to the 1.x line?

Thank you

**Comment by pilafmon** [comment]
Oops, in my haste to switch over to TypeScript, I got fast and furious on the version numbers.  My bad.

Release **v1.5.6** temporarily adds back the `module.exports` line.

However, for CommonJS/UMD support you'll want to switch over to the new **main** file at:
```
dist/clabe.umd.cjs
```

If you want to be really robust in your build process, use the `"main"` entry in **package.json** to automatically select the correct file:
https://github.com/center-key/clabe-validator/blob/master/package.json#L7
```json
   "main": "dist/clabe.umd.cjs",
```

**Comment by pilafmon** [comment]
By the way, when you upgrade to ES6 modules you'll need to switch to:
```
dist/clabe.esm.js
```

**Comment by koresar** [comment]
Thanks for the quick fix.

Unfortunately, we can't use the ESM. We are running in Node.js.
But thank you for changing "main" to be the UMD build. This is exactly what backward compatibility is needed. :)

(You're fast!)

**Comment by koresar** [comment]
@dpilafian the freshly published 1.6.0 is also SemVer breaking change.

How to reproduce.

1. click here https://npm.runkit.com/clabe-validator
2. add second line to the source code on the left: `console.log(clabeValidator.validate("123"))`
3. Click "Run"
Expected: Should print `false`
Actual: Exception `TypeError: clabeValidator.validate is not a function`

![image](https://user-images.githubusercontent.com/1622575/107733009-fed1be00-6d4d-11eb-942e-721c76dcb8d8.png)

Please, stop breaking builds. :)))

**Comment by koresar** [comment]
You might want to add a unit test like this:

```js
it("should load using CommonJS", () => {
  const clabeValidator = require("..");
  const result = clabeValidator.validate("123");
  assert.strictEqual(result.ok, false);
});
```

**Comment by pilafmon** [comment]
It works correctly:
<img width="1427" alt="runkit" src="https://user-images.githubusercontent.com/119555/107735804-6a1b8180-6cb5-11eb-8abb-678ba3613f6b.png">
You can also verify it locally right in node quite easily:
```js
% node
Welcome to Node.js v14.15.4.
Type ".help" for more information.
> const { clabe } = require('clabe-validator');
> console.log(clabe.version);
1.6.0
> console.log(clabe.validate('123'));
{
  ok: false,
  error: 'invalid-length',
  formatOk: false,
  message: 'Must be exactly 18 digits long',
  tag: null,
  bank: null,
  city: null,
  account: '',
  code: { bank: '123', city: '' },
  checksum: null
}
```
By the way, node has supported ES6 Modules for a while.

**Comment by koresar** [comment]
No, mate. You did a BREAKING change in a SemVer MINOR version (1.6.0).

Here is how you can reproduce.
1. Click here: https://npm.runkit.com/clabe-validator
2. Paste this:
```js
var clabeValidator = require("clabe-validator@1.5.1");
console.log("1.5.1", typeof clabeValidator.validate);

var clabeValidator = require("clabe-validator@1.6.0");
console.log("1.6.0", typeof clabeValidator.validate)
```
3. click "Run".

![image](https://user-images.githubusercontent.com/1622575/107743188-8c200d00-6d64-11eb-9eb5-b14af28f2312.png)

Please, get yourself familiarised with Semantic Versioning: https://semver.org/
TL;DR: API breaking changes are allowed only if you increate the **first digit** of your package version. But you increased the second digit (5->6).

Please, reopen the issue.
